WordPressVIPMinimum/Sniffs/SVG/HTMLCodeSniff.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\SVG;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * This function generates an error when
+ * <?php or <? is found in a SVG file.
+ *
+ * The aim is to bring this to the notice of
+ * reviewers, so they can make sure the PHP-code
+ * is safe. The reason for this is that SVG files
+ * are often included in PHP files, but manually
+ * scanning SVG files takes alot of time and is prone
+ * to errors.
+ */
+class HTMLCodeSniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * We want everything function-related.
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return array(
+			T_OPEN_TAG,
+		);
+	}
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file where the token was found.
+	 * @param int                         $stackPtr  The position in the stack where
+	 *                                               the token was found.
+	 *
+	 * @return void
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+		// Get tokens.
+		$tokens = $phpcsFile->getTokens();
+
+		$nxt = $phpcsFile->findNext(
+			T_OPEN_TAG,
+			( $stackPtr ),
+			null,
+			false,
+			null,
+			true
+		);
+
+		$tokens[ $nxt ]['content'] =
+			strtolower( $tokens[ $nxt ]['content'] );
+
+		$found1 =
+			strpos( $tokens[ $nxt ]['content'], '<?php' );
+
+		$found2 =
+			strpos( $tokens[ $nxt ]['content'], '<?' );
```

Comment:
Unless short open tags are allowed in the PHP config on the server where this runs, those won't be catched by `T_OPEN_TAG`.

Only way to catch short open tags is to have short open tags allowed on the server. But it only makes sense in case it mirrors the production config - I believe that short open tags are disallowed on both, WPCOM and VIP GO now.

Here's a related issue in PHPCS covering the topic of short open tags and it's catching: https://github.com/squizlabs/PHP_CodeSniffer/issues/1571 and here's a direct link to how it's handled in the Generic.PHP.DisallowShortOpenTag sniff: https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/Generic/Sniffs/PHP/DisallowShortOpenTagSniff.php#L32,L34

Reply:
I think we will let it suffice to look for `<?php` and `<?=` because of how the configuration is on our servers.

Reply:
And, to add: this test will be in conjunction with the `Generic.PHP.DisallowShortOpenTag` sniff.